src/civis/utils/_jobs.py

```diff
@@ -55,40 +78,62 @@ def run_template(id, arguments, JSONValue=False, client=None):
     arguments: dict
         Dictionary of arguments to be passed to the template.
     JSONValue: bool, optional
-        If True, will return the JSON output of the template.
-        If False, will return the file ids associated with the
-        output results.
+        Deprecated. Use 'return_as' instead.
```

Comment:
We can still keep the original doc for what `JSONValue` was meant to do, while adding the deprecation note and formatting it for the Sphinx doc page. (We're going to make a v2.8.0 release in early December.)

```suggestion
        If True, will return the JSON output of the template.
        If False, will return the file ids associated with the
        output results.

        .. deprecated:: 2.8.0
            ``JSONValue`` will be removed at civis-python v3.0.0.
            Please use ``return_as`` instead.
```